blog/articles/quickstart-with-knative.md

```diff
@@ -0,0 +1,57 @@
+# Knative Quickstart plugin
+
+---
```

Comment:
```suggestion
---
```

I think first line needs to be the metadata starting with `---`

Reply:
:facepalm: